proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -173,6 +179,10 @@ message RegisterWorkerInWorkerDeploymentArgs {
     temporal.api.enums.v1.TaskQueueType task_queue_type = 2;
     int32 max_task_queues = 3;
     WorkerDeploymentVersion version = 4;
+    
+    // populate testHook values as needed
+    int32 testhook_task_queues_sync_batch_size = 5;
```

Comment:
I don't think you should embed the testhook concept here. Can't this just be `sync_batch_size`? ie make the special case the regular case. That way the caller can set it to 25 by default, or override it.